### DISCUSSION TITLE
Program doesn't stop

### DISCUSSION THREAD

**Discussion by AndyDecker** [eveningkid/denodb#285] [Q&A]
Hello,
when i'm trying out the example from the readme the program doesn't stop, even though await db.close() is called at the end. I couldn't find any explanation for that behavior so i wondered if it is environment specific because i already used denodb a month ago on another system (which i currently don't have access to) and it worked fine.
I'm using the latest postgres version from dockerhub, ubuntu 20.04 and deno:
deno 1.12.2 (release, x86_64-unknown-linux-gnu)
v8 9.2.230.14
typescript 4.3.5
Thanks for any help and ideas!
Best Regards,
Andreas

**Comment by DavidBonelo** [comment]
I thought that was on purpose🤔? because your program may not necessarily need to end when you close the connection with the database.
I use this: https://doc.deno.land/builtin/stable#Deno.exit

**Reply by AndyDecker** [reply]
well with deno 11.5 it still closed. And deno.exit isn't useful, since i also use some database calls for testing, thus deno.exit would end the test in a failure. But thanks for the idea ;)
